spyder/config/main.py

```diff
@@ -63,6 +63,7 @@
               'single_instance': True,
               'open_files_port': OPEN_FILES_PORT,
               'tear_off_menus': False,
+              'mac_open_file': False,

```

Comment:
Should we only add this to the dictionary on mac  @ccordoba12 ?

Reply:
I don't understand what you mean by this.

Reply:
This option oNly makes sense for mac. Should we only add this option on mac or on all OSs is ok?

Reply:
I think it's ok to leave it for all OSes for now.